miio/device.py

```diff
@@ -269,7 +269,10 @@ def send(self, command: str, parameters: Any=None, retry_count=3) -> Any:
                           m.data.value["id"],
                           m.data.value)
             if "error" in m.data.value:
-                raise DeviceError(m.data.value["error"])
+                error = m.data.value["error"]
+                if "code" in error and error["code"] == -30001:
+                    raise RecoverableError(error)
```

Comment:
We want to have this codepath tested too (at some point, maybe there are also other recoverable cases?) :-)

Reply:
Do you've an idea how to test this part?

Reply:
I think the way to go would be mocking the Message.parse() to return with that error code, and then checking that it gets  called retry_count times. #437 may simplify the tests (or we might split the code even a bit more to make it easier to test). I think this can wait for now, feel free to merge.